### PR TITLE
Add support for exporting CMS templates

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/qrda-export/catIII-r2-1/qrda3_r21.rb [hunk 1]
```diff
@@ -6,6 +6,8 @@ class Qrda3R21 < Mustache
 
   self.template_path = __dir__
 
+  PAYER_MAP = { '1' => 'A', '2' => 'B', '349' => 'D' }.freeze
```
Comment (mayerm94) [inline lib/qrda-export/catIII-r2-1/qrda3_r21.rb]: Aren't there hundreds of payer codes? How can we be sure that only these 3 will be used?
